lib/serializer.ts

```diff
@@ -231,7 +235,9 @@ class ResolvedDeclarationEmitter {
         }
       }
 
-      let output = children.map(n => this.emitNode(n)).join('');
+      let output = children.filter(n => n.kind !== ts.SyntaxKind.FirstJSDocTagNode)
```

Comment:
In newer TypeScript JSDoc comments are parsed into their own node. Since `ts-api-guardian` is supposed to strip all doc comments we can simply skip them in the output.